### ISSUE TITLE
Weights for the Behavior Models?

### ISSUE THREAD

**Issue by philippe-eecs** [ChenDRAG/CEP-energy-guided-diffusion#1]
Hi, Could you provide the pre-trained behavior models weights for the locomotion and antmaze environments? This would save me a lot of time for an experiment I need to run!

Thanks

**Comment by philippe-eecs** [comment]
I've collected them myself. Thanks.

**Comment by ChenDRAG** [comment]
So sorry for the late response, I will upload the behavior model in two days.

**Comment by philippe-eecs** [comment]
No worries, I've collected the weights I needed for an experiment sweep already

**Comment by ChenDRAG** [comment]
Hi, Philippe @philippe-eecs . I've already uploaded the checkpoints for toy and rl experiments (both behavior and the critic guidance model) just in case you might still need it for debugging or something. There is only one seed though.

URL : https://drive.google.com/drive/folders/1snFcmcJaalcCWW9roBjeCjpWjpCeDM_P?usp=drive_link

I did not test them for now. Tell me if you encounter any difficulties reproducing the experiment. 

I will close this issue for now (e4835e2).
